apps/trading-e2e/src/integration/capsule.cy.ts

```diff
@@ -116,26 +134,144 @@ describe('capsule', { tags: '@slow' }, () => {
         });
         checkIfDataAndTimeOfCreationAndUpdateIsEqual(orderUpdatedAt);
       });
-
+    //cancel order
+    cy.getByTestId(ordersTab).click();
     cy.getByTestId('cancel').first().click();
-
-    cy.getByTestId('dialog-title').should(
+    cy.getByTestId(toastContent).should(
       'contain.text',
-      'Awaiting network confirmation'
+      `ConfirmedYour transaction has been confirmed View in block explorerCancel order - cancelledTEST.24h+0.0005 @ 200.00 ${usdcSymbol}`
     );
-    cy.getByTestId('dialog-title').should('contain.text', 'Order cancelled');
-    cy.getByTestId('dialog-close').click();
+    cy.getByTestId(toastCloseBtn).click();
 
     cy.getByTestId('tab-orders')
       .get('.ag-center-cols-container')
       .children()
       .first()
-      .get(`[col-id='${orderStatus}']`)
+      .get(`[col-id='${orderStatus}']`, txTimeout)
       .should('contain.text', OrderStatusMapping.STATUS_CANCELLED);
   });
+
+  it('can deposit and withdrawal', function () {
```

Comment:
suggestion (non blocking): Can we split this into two separate tests one for deposit and one for withdrawal. We should probably also do the same for order submit, edit and cancel. I think even though each test relies on the previous test it should still result in better reporting if the test fails

Reply:
it was done this way to save some time as we have dependent capsule tests anyway. I will check how the execution time will change after separating them and depending on the results we will decide what to do.